src/client/Contracts.js

```diff
@@ -51,6 +51,33 @@ exports.ContractAbi = async function({contractAddress, id}) {
   return contractInfo.abi;
 };
 
+/**
+ * Retrieve the ABI, access type, and whether V3 is used for a given contract via its address or a Fabric ID.
+ *
+ * @methodGroup Contracts
+ * @namedParams
+ * @param {string=} id - The Fabric ID of the contract
+ * @param {string=} address - The address of the contract
+ *
+ * @return {Promise<Object>} - The ABI, access type, and isV3 for the given contract
+ */
+exports.ContractInfo = async function({id, address}) {
```

Comment:
what I meant by a proxy method is that it should just call the existing method and return it. no need for a full re-implementation

just do `return this.authClient.ContractInfo({id, address});`

Reply:
the main reason this is needed is because frame client doesn't have access to auth client, so this bridges that gap